src/signers/EdDSASigner.ts

```diff
@@ -18,8 +18,8 @@ import { bytesToBase64url, stringToBytes } from '../util.js'
  */
 export function EdDSASigner(secretKey: Uint8Array): Signer {
   const privateKeyBytes: Uint8Array = secretKey
-  if (privateKeyBytes.length !== 64) {
-    throw new Error(`bad_key: Invalid private key format. Expecting 64 bytes, but got ${privateKeyBytes.length}`)
+  if (![32, 64].includes(privateKeyBytes.length)) {
```

Comment:
any chance you could add a comment that points to a resource explaining that both key lengths are acceptable?

Reply:
Of course